charts/openyurt/crds/apiextensions.k8s.io_v1_customresourcedefinition_gateways.raven.openyurt.io.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
how about rename file name `apiextensions.k8s.io_v1_customresourcedefinition_gateways.raven.openyurt.io.yaml` to `gateways.raven.openyurt.io.yaml`? because i think that `apiextensions.k8s.io_v1_customresourcedefinition` is a useless info.